picachu/modules/galleries/galleries_routers.py

```diff
@@ -81,10 +85,43 @@ def get_galleries():
         return jsonify({'msg': 'Forbidden'}), HTTPStatus.FORBIDDEN
     try:
         list_galleries = GetGalleriesQuery().get(current_user_id)
-        print(list_galleries)
         result = []
         for gallery in list_galleries:
             result.append({'id': gallery.id, 'name': gallery.name, 'photosCount': GetPhotoQuery.count_photos(gallery.id)})
         return result
     except Exception as err:
         return jsonify(str(err)), HTTPStatus.BAD_REQUEST
+
+
+@galleries_blueprint.route('/<int:gallery_id>/photos', methods=['GET'])
+@jwt_required()
+def get_photos(gallery_id):
+    current_user_id = get_jwt_identity()
+    if not IsUserHasAccess.to_gallery(current_user_id, gallery_id):
+        return jsonify({'msg': 'Forbidden'}), HTTPStatus.FORBIDDEN
+    if not GetGalleryQuery.by_id(gallery_id):
+        return jsonify({'msg': 'Not Found'}), HTTPStatus.NOT_FOUND
+
+    params = SortingParams(offset=request.args.get('offset'),
+                           limit=request.args.get('limit'),
+                           sorted_by=request.args.get('sortedBy'))
+    photos_sorted = GetSortedPhotosQuery().get_sorted_photos(gallery_id,
+                                                             params.sorted_by,
+                                                             params.offset,
+                                                             params.limit)
+    result = []
+    try:
+        for photo in photos_sorted:
+            result.append(
+                {
+                    'id': photo.id,
+                    'photo_file_path_s3': photo.photo_file_path_s3,
+                    'uniqueness': random.randint(0, 100)
```

Comment:
After I made the changes and added uniqueness generation and writing it to the database, here I need to return the uniqueness read from the database. Another query is needed.

Reply:
Do I need to pull changes from develop?